protoc-gen-twirp_ruby/main.go

```diff
@@ -273,3 +281,12 @@ func isASCIILower(c byte) bool {
 func isASCIIDigit(c byte) bool {
 	return '0' <= c && c <= '9'
 }
+
+// RubyPackageParser can parse the ruby package option added in protobuf v3.6.0 but not yet in golang/protobuf
```

Comment:
> not yet in golang/protobuf

this code should not concern with the version of `golang/protobuf` or `protocolbuffers/protobuf` that is being used, because the version may change at anytime (and the comment would be worthless). Maybe a better way to describe this struct is that the `Package` property will contain the ruby_package option if the protobuf compiler supports it. I would just comment the field like this:

```go
// RubyPackageParser struct to use with proto.Unmarshal and the file descriptor. 
type RubyPackageParser struct {
	// RubyPackageOption is the ruby_package option in the proto file.
	// It is empty if there is no option, or if not supported by the proto package
	// This option was added in protocolbuffers/protobuf v3.6.0+
	OptionRubyPackage string `protobuf:"bytes,45,opt,name=ruby_package,json=rubyPackage,proto2" json:"ruby_package,omitempty"`
}
```